nb_cli/handlers/meta.py

```diff
@@ -77,13 +77,16 @@ async def get_default_python() -> str:
             )
             stdout, stderr = await proc.communicate()
             if proc.returncode == 0:
-                if stdout:
-                    return json.loads(stdout.strip())
-                else:
-                    raise PythonInterpreterError(
-                        _(f"Cannot get valid `sys.executable`, got {stdout}.")
-                    )
-        raise PythonInterpreterError(_("Cannot find a valid Python interpreter."))
+                try:
+                    if executable := json.loads(stdout.strip()):
+                        return executable
+                except Exception:
+                    continue
+        raise PythonInterpreterError(
+            _(
+                f"Cannot find a valid Python interpreter. stdout={stdout}, stderr={stderr}"
+            )
+        )
```

Comment:
stderr没有pipe会直接输出到控制台

```suggestion
        raise PythonInterpreterError(_("Cannot find a valid Python interpreter."))
```